infra_provision/docs/HostactivationJoinToken.md

```diff
@@ -19,7 +19,7 @@ Name | Type | Description | Notes
 
 ### NewHostactivationJoinToken
 
-`func NewHostactivationJoinToken() *HostactivationJoinToken`
+`func NewHostactivationJoinToken(name string, ) *HostactivationJoinToken`
```

Comment:
This change is unexpected for this PR. Also name is supposed to be required. But previously it was generated with name as optional. Having this change will require the terraform provider to be fixed as well.

Reply:
As discussed, also update Terraform Provider.